### PR TITLE
Add csepanyij's pin to map

### PR DESCRIPTION
**Pull request by csepanyij**
Adding Budapest to the map!

### CODE REVIEW

File: _pins/csepanyij.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+githubHandle:csepanyij
```
Comment (brianamarie) [inline _pins/csepanyij.json]: Try adding a line of three hyphens as line one (like this: `---`), and then have githubHandle on line 2.

Comment (brianamarie) [inline _pins/csepanyij.json]: Make sure there is a space after the colon and before `csepanyij`, and follow that pattern for latitude and longitude.

File: _pins/csepanyij.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+latitude:47.497912
+logitude:19.040235
+
```
Comment (brianamarie) [inline _pins/csepanyij.json]: Add the same `---` on the last line, too.

File: _pins/csepanyij.json (hunk 1 above)
Comment (csepanyij) [inline _pins/csepanyij.json]: Sorry, I forgot about these, I think I got it right now.

**Comment by tuffix** [comment]
Hej @csepanyij try changing logitude to longitude! I think that should work :)

**Comment by tuffix** [comment]
Hej @csepanyij,
Missed this last time but ad a space, after githubHandle: csepanyij think it should work than!
Good luck!!

**Comment by crichID** [comment]
@easpaas please do not close pull requests and issues that you did not create! We know it is tempting, but we want everyone to have the opportunity to finish the project on their own. If you are interested in helping, then leaving supporting comments is :sparkles: and encouraged.

**Pull request merged**